views/default/object/elements/summary.php

```diff
@@ -44,15 +44,12 @@
 
 $icon = elgg_extract('icon', $vars);
 if (isset($icon)) {
-	$params = $vars;
-	foreach (['title', 'metadata', 'subtitle', 'tags', 'content', 'icon', 'class'] as $key) {
-		unset($params[$key]);
-	}
 	$class = (array) elgg_extract('class', $vars, []);
 	$class[] = 'elgg-listing-summary';
-	$params['class'] = $class;
-
-	$params['data-guid'] = $entity->guid;
+	$params = [
```

Comment:
Should we build this from `elgg_extract('image_block_attrs', $vars, [])`? That would leave the ability to specify it.